content/docs/en/elements/layouts/grid-layout.md

```diff
@@ -49,25 +58,39 @@ The GridLayout is a layout that arranges its child elements in a table structure
   <Label text="1,1" row="1" col="1" backgroundColor="#43b883"/>
 </GridLayout>
 ```
-<img class="md:w-1/2 lg:w-1/3" src="https://art.nativescript-vue.org/layouts/grid_layout_fixed_auto.svg" />
+<img class="md:w-1/2 lg:w-1/3" src="https://art.nativescript-vue.org/layouts/grid_layout_complex.svg" />
```

Comment:
This has been changed by accident, should be `https://art.nativescript-vue.org/layouts/grid_layout_fixed_auto.svg`